dali/base/daclient.cpp

```diff
@@ -88,11 +88,11 @@ IDaliClient_Exception *createClientException(DaliClientError err, const char *ms
 
 
 
-bool initClientProcess(IGroup *servergrp, DaliClientRole role, unsigned mpport, const char *clientVersion, const char *minServerVersion, unsigned timeout)
+bool initClientProcess(IGroup *servergrp, DaliClientRole role, unsigned mpport, const char *clientVersion, const char *minServerVersion, unsigned timeout, bool _listen)
 {
     assertex(servergrp);
     daliClientIsActive = true;
-    startMPServer(role, mpport);
+    startMPServer(role, mpport, false, _listen);
```

Comment:
trivial: no reason for this param to be prefixed with _ I think.

Reply:
done

Reply:
fixed